core/cmd/remote_client.go

```diff
@@ -466,10 +466,11 @@ func (cli *Client) ConfigDump(c *clipkg.Context) (err error) {
 func (cli *Client) ConfigFileValidate(c *clipkg.Context) error {
 	err := cli.Config.Validate()
 	if err != nil {
-		return err
+		fmt.Println("Invalid configuration:", err)
+		fmt.Println()
 	}
 	cli.Config.LogConfiguration(func(params ...any) { fmt.Println(params...) })
-	return err
+	return nil
```

Comment:
So this will never return an error, why?

Reply:
Because the user-facing error is already logged above, and there are incidentally no internal application errors here to pass along since all of the validation happens on in-memory structs. If we also return the user facing error here, then callers up the stack make assumptions about it being a severe problem, as well as double log things.

I could see a case being made for returning _something_ to trigger a non-zero exit code at least, but that still seems debatable, since the command itself is executing successfully.

Reply:
So I wonder if we should change the signature of this function, to not return an error at all, unless we are complying to some interface here.

Reply:
Yeah I think it matches an expected signature.